File: MessDotCity/src/app/_models/dailyExpense.ts
export interface DailyExpense{
    id?: number;
    day: any;
    expense: number;
    messId?: number;
    responsibleMember?: string;
    totalMeal: number;
}

export interface MealForMember {
    memberId: number;
    memberName: string;
    breakfast: number;
    lunch: number;
    dinner: number;
    photoUrl: string;
}

File: MessDotCity.API/Models/DailyExpense.cs
using System;

namespace MessDotCity.API.Models
{
    public class DailyExpense
    {
        public int Id { get; set; }
        public DateTime Day { get; set; }
        public int MessId { get; set; }
        public float Expense { get; set; }
        public string ResponsibleMember { get; set; }
        public float TotalMeal  { get; set; }
    }
}